ziggurat_form/form.py

```diff
@@ -18,9 +22,15 @@ def __init__(self, schema_cls, bind_values=None, after_bind_callback=None):
         self.bind_values = bind_values
         self.after_bind_callback = after_bind_callback
         self.valid = None
-        self.schema_instance = self.schema_cls(after_bind=self.after_bind_callback)
+        if type(self.schema_cls) is colander._SchemaMeta:
+            self.schema_instance = self.schema_cls(
+                after_bind=self.after_bind_callback)
+        else:
+            # if it's colander.SchemaNode for example
+            self.schema_instance = self.schema_cls
```

Comment:
This line is allow load scheme from ColanderAlchemy.

![sacrud_-_update_mptt_pages_-_2015-10-28_17 54 09](https://cloud.githubusercontent.com/assets/619015/10789804/6d8b3cf6-7da1-11e5-84f8-80194ff6da6c.png)